bigtable/admin/table_admin.cc

```diff
@@ -0,0 +1,74 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/admin/table_admin.h"
+
+#include <thread>
+
+namespace btproto = ::google::bigtable::admin::v2;
+
+namespace bigtable {
+inline namespace BIGTABLE_CLIENT_NS {
+std::string TableAdmin::CreateInstanceName() const {
+  std::string result("projects/");
```

Comment:
Please use `abseil::StrCat` here instead, without a temporary var.

Reply:
That requires compiling `abseil`.  Let me take a look, but it is not trivial.

Reply:
Done.  That function better be **amazing** fast, because it drags an interesting set of dependencies.